metadata_service/app.py

```diff
@@ -17,99 +17,60 @@
 
 logger = logging.getLogger()
 
-app = Flask(__name__)
-app.register_blueprint(observability)
-app.register_blueprint(metadata_api)
+app = FastAPI()
+app.include_router(observability_router)
+app.include_router(metadata_router)
 
 setup_logging(app)
+setup_uvicorn_logging()
 
 
-@app.after_request
-def after_request(response: Response):
-    if (
-        "Accept" in request.headers
-        and request.headers["Accept"] == "application/x-msgpack"
-    ):
-        # create a new Response to send the payload only as "data" field
-        response_msgpack = make_response(msgpack.dumps(response.json))
-        response_msgpack.headers.set("Content-Type", "application/x-msgpack")
-        return response_msgpack
-
+@app.middleware("http")
+async def add_language_header(request, call_next):
+    response = await call_next(request)
+    response.headers.setdefault("Content-Language", "no")
     return response
 
 
-@app.errorhandler(Exception)
-def handle_generic_exception(exc):
+@app.exception_handler(Exception)
+def handle_generic_exception(_req, exc):
     logger.exception(exc)
-    return (
-        jsonify(
-            {
-                "code": 202,
-                "message": f"Error: {str(exc)}",
-                "service": "metadata-service",
-                "type": "SYSTEM_ERROR",
-            }
-        ),
-        500,
-    )
-
-
-@app.errorhandler(NotFound)
-def handle_url_invalid(exc):
-    logger.warning(exc, exc_info=True)
-    return (
-        jsonify(
-            {
-                "code": 103,
```

Comment:
I don't see the 103 code moved anywhere. We need to keep the follwing contract: https://gitlab.sikt.no/raird/data-store-api-doc#error-responses

Reply:
I found a place in the code where they use this 👍🏻 Adding the 103 back